src/cljs/main/broadfcui/page/workspace/notebooks/notebooks.cljs

```diff
@@ -0,0 +1,330 @@
+          [:div {:style {:marginBottom -20}}
+           (when deleting? [comps/Blocker {:banner "Deleting cluster..."}])
+           [:div {} (str "Are you sure you want to delete cluster " cluster-to-delete "?")]
+           [comps/ErrorViewer {:error server-error}]])}]))
+   :-delete-cluster
+   (fn [{:keys [this state refs props]}]
+     (let [{:keys [cluster-to-delete]} props]
+       (swap! state assoc :deleting? true)
+       (swap! state dissoc :server-error)
+       (endpoints/call-ajax-leo
+        {:endpoint (endpoints/delete-cluster (get-in props [:workspace-id :namespace]) cluster-to-delete)
+         :headers utils/content-type=json
+         :on-done (fn [{:keys [success? get-parsed-response]}]
+                    (swap! state dissoc :deleting?)
+                    (if success?
+                      (do ((:dismiss props)) ((:reload-after-delete props))) ;if success, update the table?
+                      (swap! state assoc :server-error (get-parsed-response false))))})))})
+
+
+(react/defc- NotebooksTable
+  {
+   :render
+   (fn [{:keys [state props this]}]
+     (let [{:keys [clusters toolbar-items]} props]
+       [:div {}
+        (when (:show-delete-dialog? @state) [ClusterDeleter (assoc props :dismiss #(swap! state dissoc :show-delete-dialog?)
+                                                                         :cluster-to-delete (:cluster-to-delete @state))])
+        [Table
+         {:data clusters
+          :body {:empty-message "There are no clusters to display."
+                 :style table-style/table-light
+                 :behavior {:reorderable-columns? true}
+                 :columns
+                 [{:header "" :initial-width 30
+                   :resizable? false :sortable? false :filterable? false
+                   :as-text :clusterName :sort-by :clusterName
+                   :render
+                   (fn [cluster]
+                     (if (= (:status cluster) "Running")
+                       [:a {:style {:color (:text-light style/colors)
+                                    :minHeight 30 :minWidth 30}
+                            :href "javascript:;"
+                            :onClick (fn [] (swap! state assoc :show-delete-dialog? true :cluster-to-delete (:clusterName cluster)))
+                            :id (:id props) :data-test-id "x-button"}
+                        (icons/render-icon {} :delete)]))}
+
+                  {:header "Name" :initial-width 200
+                   :as-text :clusterName :sort-by :clusterName :sort-initial :asc
+                   :render
+                   (fn [cluster]
+                     (if (= (:status cluster) "Running")
+                       [:a {:style {:textDecoration "none"}
+                            :href (str (config/leonardo-url-root) "notebooks/" (:googleProject cluster) "/" (:clusterName cluster))
+                            :onClick #(utils/set-notebooks-access-token-cookie (utils/get-access-token))
```

Comment:
This sets the token in the new tab, the token will be auto-refreshed using a Notebooks extension